src/scss/_sofie-utility-classes.scss

```diff
@@ -0,0 +1,251 @@
+.m-0 {
+  margin: 0;
+}
+
+.m-1 {
+  margin: 4px;
+}
+
+.m-2 {
+  margin: 8px;
+}
+
+.m-3 {
+  margin: 12px;
+}
+
+.m-4 {
+  margin: 20px;
+}
+
+.m-5 {
+  margin: 30px;
+}
+
+.ms-0 {
+  margin-left: 0;
+}
+
+.ms-1 {
+  margin-left: 4px;
+}
+
+.ms-2 {
+  margin-left: 8px;
+}
+
+.ms-3 {
+  margin-left: 12px;
+}
+
+.ms-4 {
+  margin-left: 20px;
+}
+
+.ms-5 {
+  margin-left: 30px;
+}
+
+.me-0 {
```

Comment:
ms- margin-start, me- margin-end. This is a new trend after the introduction of flex box from where we have flex-end and flex-start. Bootstrap and now I see places in Angular Material use that naming conventions. I think it was introduced because of possible element rotation and then left is no longer left and right is right